recipes/date/all/conanfile.py

```diff
@@ -24,13 +24,15 @@ class DateConan(ConanFile):
         "header_only": [True, False],
         "use_system_tz_db": [True, False],
         "use_tz_db_in_dot": [True, False],
+        "manual_tz_db": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "header_only": False,
         "use_system_tz_db": False,
         "use_tz_db_in_dot": False,
+        "manual_tz_db": False,
     }
```

Comment:
Please add an `options_description = { "use_system_tz_db": "...", ...}` attribute to the recipe. Their meaning of these options and how they interact is a bit vague otherwise.

I would also maybe call the new option `allow_tz_db_download` instead.

Reply:
Done